.rubocop.yml

```diff
@@ -0,0 +1,32 @@
+AllCops:
+  Exclude:
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'log/**/*'
+    - 'tmp/**/*'
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 150
+
+Metrics/AbcSize:
+  Max: 20
```

Comment:
why 20?

Reply:
Cause ABC size is a very abstract parameter for code quality assessment and based to my experience, if it will may more longer methods - code will be cleaner, without unnecessary abstraction. By the way by default it is 15.

Reply:
and what's the problem with 15?

Reply:
Take a look at this [prepare_params](https://github.com/rubizza-camp/summer-2019/blob/master/3444/1/lib/top-gems/stats_of_gem_parser.rb) method from my home task, it's looking much better if i'm using such structure, according to rubocop i separated it before into few abstract methods and it was really ugly.
If it's so really impossible, of cause i will resolve it

Reply:
hmm, your code with or without looks really ugly, if we speak in these terms) 
if you want to prove some, refer to smth more reliable, than you own code, bro

Reply:
no matter

Reply:
http://jakescruggs.blogspot.com/2008/08/whats-good-flog-score.html

Reply:
https://codeclimate.com/blog/deciphering-ruby-code-metrics/

Reply:
All of them and others relying that it's possible to score upto 20.